genome_designer/utils/data_export_util.py

```diff
@@ -181,7 +185,8 @@ def export_var_dict_list_as_vcf(var_dict_list, vcf_dest_path_or_filehandle,
                 1, # QUAL
                 [], # FILTER
                 {
-                    'METHOD': 'DE_NOVO_ASSEMBLY'
+                    'METHOD': 'DE_NOVO_ASSEMBLY',
+                    'CALLER': caller_string
```

Comment:
Yeah I had been adding SV discovery things under the DE_NOVO_ASSEMBLY INFO_METHOD, because it has been convenient to be able to get all the variants coming from any of the SV discovery strategies and then I made the CALLER field to distinguish between the different strategies.  

Now that INFO__METHO DE_NOVO_ASSEMBLY includes read-verified contig placement, sequence graph walks, and coverage based deletion detection it's kind of a misnomer, so I'll just make them
INFO__METHOD DE_NOVO_ASSEMBLY,
INFO__METHOD  GRAPH_WALK, and
INFO__METHOD_COVERAGE respectively.